QUEST_LV_0100.tsv

```diff
@@ -3823,139 +3823,139 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
+QUEST_LV_0100_20150317_003938	$Seems like the Supply Officer is in trouble. Try to talk to him.
 QUEST_LV_0100_20150317_003939	{memo X}Move to the marked location
 QUEST_LV_0100_20150317_003940	{memo X}Track the trails of the Weaver in the marked area.
 QUEST_LV_0100_20150317_003941	{memo X}Defeat Weaver and report to the Supply Officer.
-QUEST_LV_0100_20150317_003942	Defeat Weaver
-QUEST_LV_0100_20150317_003943	Talk to Aras
-QUEST_LV_0100_20150317_003944	Talk about what to do now to go to Crystal Mine
-QUEST_LV_0100_20150317_003945	Collect clues from Popolions
-QUEST_LV_0100_20150317_003946	To enter Crystal Mine as promised with Aras, defeat Popolions and find some clues why the number of monsters has increased.
-QUEST_LV_0100_20150317_003947	Give piece of Vubbe's cloth to Operations Officer
-QUEST_LV_0100_20150317_003948	Found piece of Vubbe cloth. Can this be a clue? Give it to the Operations Officer
-QUEST_LV_0100_20150317_003949	Talk to the Operations Officer
-QUEST_LV_0100_20150317_003950	The Operations Officer seems to know something with the Vubbe cloth. {nl}Try talking to him again.
-QUEST_LV_0100_20150317_003951	Investigate the Northern Areas
-QUEST_LV_0100_20150317_003952	The Operations Officer says there are used to be no Vubbes in Eatern Woods. Check the northern area of Siauliai Easter Woods while the Operations Officer ask Aras for help.
-QUEST_LV_0100_20150317_003953	Report to the Operations Officer
-QUEST_LV_0100_20150317_003954	Is the Vubbe Fighter controlling the Vubbes? Tell the Operations Officer about what you saw.
-QUEST_LV_0100_20150317_003955	Defeat Vubbe Fighter's subordinates
-QUEST_LV_0100_20150317_003956	Talk to Aras about the Vubbe Fighter.
-QUEST_LV_0100_20150317_003957	Search for Vubbe Fighter
-QUEST_LV_0100_20150317_003958	Is this one of the promises for Crytal Mines? Check how many Vubbe Fighters there are.
+QUEST_LV_0100_20150317_003942	$Defeat Weaver
+QUEST_LV_0100_20150317_003943	$Talk to Aras
+QUEST_LV_0100_20150317_003944	$Talk about what to do next in order to enter the Crystal Mine
+QUEST_LV_0100_20150317_003945	$Collect clues from Popolions
+QUEST_LV_0100_20150317_003946	$In order for Aras to permit you to enter the Crystal Mine, defeat Popolions and find some clues why the number of monsters has increased.
+QUEST_LV_0100_20150317_003947	$Give the Piece of Vubbe Cloth to the Operations Officer
+QUEST_LV_0100_20150317_003948	$Found a Piece of Vubbe Cloth. Can this be a clue? Give it to the Operations Officer
+QUEST_LV_0100_20150317_003949	$Talk to the Operations Officer
+QUEST_LV_0100_20150317_003950	$The Operations Officer seems to have realized something with the Vubbe Cloth. {nl}Try talking to him again.
+QUEST_LV_0100_20150317_003951	$Investigate the Northern Areas
+QUEST_LV_0100_20150317_003952	$The Operations Officer says there are used to be no Vubbes in the Eastern Woods. Check the northern area of the East Siauliai Woods while the Operations Officer ask Aras for help.
```

Comment:
"The Operations Officer says there used to be no Vubbes in the Eastern Woods before. Check the northern area of the East Siauliai Woods while the Operations Officer asks Aras for help."